### DISCUSSION TITLE
set directory path as the default save path

### DISCUSSION THREAD

**Discussion by Kellermaan** [gildas-lormeau/SingleFileZ#160] [Q&A]
Hi I really like your app here.
And I have question that, can I set a directory path as the default save path. So the web file can be saved there.

**Comment by Kellermaan** [comment]
Can I get any help here? Thanks in advance!

**Comment by gildas-lormeau** [comment]
Web extensions cannot save files outside the Downloads folder automatically. However, you can use the setting `Filename > template` to save files in a sub-folder of the Downloads folder.

**Comment by Kellermaan** [comment]
Ok. I didn't know it untill now. Sorry for the sily question. Thanks a lot!

**Comment by gildas-lormeau** [comment]
No problem ;)

**Comment by maliayas** [comment]
You can use symlinks for that. See https://github.com/gildas-lormeau/SingleFile/issues/823#issuecomment-1198111015
